src/instructlab/model/backends/backends.py

```diff
@@ -319,12 +319,16 @@ def get_uvicorn_config(app: fastapi.FastAPI, host: str, port: int) -> Config:
     )
 
 
-def select_backend(cfg: serve_config, backend: Optional[str] = None) -> BackendServer:
+def select_backend(
+    cfg: serve_config,
+    backend: Optional[str] = None,
+    model_path: pathlib.Path | None = None,
+) -> BackendServer:
     # Local
     from .llama_cpp import Server as llama_cpp_server
     from .vllm import Server as vllm_server
 
-    model_path = pathlib.Path(cfg.model_path)
+    model_path = pathlib.Path(model_path or cfg.model_path)
```

Comment:
I'm not going to block on this but just as an FYI you should prefer to use `cfg.model_path if not model_path else model_path` in python when doing these default evaluations.